source/scale/scale-to-100-users.rst

```diff
@@ -12,13 +12,19 @@ This page describes the Mattermost reference architecture designed for the load
 Requirements
 ------------
 
-+------------------------+----------+------------------+----------------+------------------+
-| **Resource Type**      | **vCPU** | **Memory (GiB)** | **# of Nodes** | **AWS Instance** |
-+========================+==========+==================+================+==================+
-| Mattermost Application | 2        | 4                | 1              | c6i.large        |
-+------------------------+----------+------------------+----------------+------------------+
-| RDS Writer             | 2        | 16               | 1              | db.r6g.large     |
-+------------------------+----------+------------------+----------------+------------------+
++------------------------+----------------+-------------------+
+| **Resource Type**      | **# of Nodes** | **AWS Instance**  |
++========================+================+===================+
+| Mattermost Application | 1              | c7i.large         |
++------------------------+----------------+-------------------+
+| RDS Writer             | 1              | db.r7g.large      |
++------------------------+----------------+-------------------+
+| RDS Reader             | 0              | db.r7g.large      |
++------------------------+----------------+-------------------+
+| Elasticsearch Node     | 0              | r6g.xlarge.search |
```

Comment:
Should we remove the rows with 0 nodes? Or do we prefer the consistency over all docs?